ui/css/itcss/components/send.scss

```diff
@@ -1,927 +0,0 @@
-    width: 24px;
-    border: 1px solid var(--color-primary-default);
-    border-radius: 4px;
-    background-color: var(--color-background-default);
-    position: absolute;
-    right: 15px;
-    top: 14px;
-    cursor: pointer;
-  }
-
-  &__sliders-icon {
-    color: var(--color-primary-default);
-  }
-
-  &__memo-text-area {
-    &__input {
-      padding: 6px 10px;
-    }
-  }
-
-  &__footer {
-    height: 92px;
-    width: 100%;
-    display: flex;
-    justify-content: space-evenly;
-    align-items: center;
-    border-top: 1px solid var(--color-border-muted);
-    background: var(--color-background-default);
-    padding: 0 12px;
-    flex-shrink: 0;
-  }
-
-  &__next-btn,
-  &__cancel-btn {
-    width: 163px;
-    margin: 0 4px;
-  }
-
-  &__gas-modal-card {
-    width: 360px;
-    display: flex;
-    flex-flow: column;
-    align-items: flex-start;
-    padding-left: 20px;
-
-    &__title {
-      @include design-system.H4;
-
-      height: 26px;
-      color: var(--color-text-alternative);
-      margin-top: 17px;
-    }
-
-    &__copy {
-      @include design-system.Paragraph;
-
-      height: 38px;
-      width: 314px;
-      color: var(--color-text-alternative);
-      margin-top: 17px;
-    }
-
-    .gas-tooltip-input-arrows {
-      @include design-system.H4;
-
-      width: 32px;
-      height: 54px;
-      border-left: 1px solid var(--color-border-muted);
-      color: var(--color-text-alternative);
-      right: 0;
-      padding: 0;
-      display: flex;
-      justify-content: space-around;
-      align-items: center;
-    }
-
-    .gas-tooltip-input-arrow-wrapper {
-      align-items: center;
-      align-self: stretch;
-      display: flex;
-      flex-direction: column;
-      flex-grow: 1;
-      justify-content: center;
-    }
-
-    input[type='number']::-webkit-inner-spin-button {
-      -webkit-appearance: none;
-      display: none;
-    }
-
-    input[type='number']:hover::-webkit-inner-spin-button {
-      -webkit-appearance: none;
-      display: none;
-    }
-  }
-}
-
-.advanced-gas-options-btn {
-  @include design-system.H6;
-
-  display: flex;
-  justify-content: flex-end;
-  color: var(--color-primary-default);
-  cursor: pointer;
-  margin-top: 5px;
-}
-
-.sliders-icon-container {
-  @include design-system.Paragraph;
-
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 24px;
-  width: 24px;
-  border: 1px solid var(--color-primary-default);
-  border-radius: 4px;
-  background-color: var(--color-background-default);
-  position: absolute;
-  right: 15px;
-  top: 14px;
-  cursor: pointer;
-}
-
-.gas-fee-reset {
-  @include design-system.H6;
-
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 24px;
-  border-radius: 4px;
-  background-color: var(--color-background-default);
-  position: absolute;
-  right: 12px;
-  top: 14px;
-  cursor: pointer;
-  color: var(--color-primary-default);
-}
-
-.sliders-icon {
-  color: var(--color-primary-default);
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 12 06 18 PM](https://github.com/user-attachments/assets/3a0e1d69-0660-4d74-ba1e-bb4351e13d70)